tritonbench/operators/gdpa/gdpa.py

```diff
@@ -158,7 +156,6 @@ def _gdpa_fwd_inner_ws(
             p = qk
 
         p *= qk_scale
-        p = p.to(v_dtype)
```

Comment:
Deleted this type cast since it has a duplicate after the next if-else statement.